content/foundations/getting-started.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Getting started
+---
+
+import {Box} from '@primer/react'
+
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  Primer is built for GitHub by GitHub. Its guidelines are a collection of open source principles, standards, and recommendations for designing GitHub. As a design language, Primer provides common grammar and vocabulary to build cohesive, accessible, responsive, and efficient experiences.
+</Box>
+
+## A cohesive experience
+
+GitHub has a global audience. Millions of people use GitHub every day for many different reasons, in many different ways. They all come from different backgrounds. We can’t infer their physical or cognitive abilities, or devices they use, or the internet connection they have. All of that needs to be taken into account before, during, and after the design process.
+
+Familiar patterns help people intuitively navigate to the information they're looking for, while also helping people discover new features. If pages and features don’t share the same metaphors, people get confused more easily, it makes it harder for them to understand the features we provide.
+
+Primer focuses on the user experience so people can benefit from using a platform that is cohesive, no matter where in the platform they are. If a page behaves and feels familiar, it allows people to focus on the task at hand, instead of requiring them to parse UI that looks completely new.
+
+## Inclusive and accessible design
+
+GitHub is a home for all developers. To be inclusive means we must consider accessibility at the core of how we design.
+
+Accessibility and inclusive design needs to be expected from the start of a project, because it deeply affects its design, how the product behaves, and how it’s built.
```

Comment:
```suggestion
Accessibility and inclusive design needs to be considered from the start of a project, because it deeply affects its design, how the product behaves, and how it’s built.
```